### ISSUE TITLE
Anyone know if its possible to play audio over a camera?

### ISSUE THREAD

**Issue by developersteve** [twrecked/hass-aarlo#688]
I couldnt see it listed anywhere but it would be great to be able to play audio on a camera as a response to certain events, person detected etc.

**Comment by twrecked** [comment]
We can't do it at the moment. But, if we can use the microphone from the web interface we should be able to use it in Home Assistant. I'll take a look.

**Comment by developersteve** [comment]
Yup its available, this would take hassio security to a whole other level being able to play automated sound back through the camera ... oh the fun to be had 😄

**Comment by developersteve** [comment]
So we just need the endpoint and audio format?

**Comment by twrecked** [comment]
It's maybe a little more complicated than that but not much. There are no fixed endpoints to the cameras with _Arlo_, you normally have to request them as needed.

If you open the browser developer tools (usually `CTRL+SHIFT+I`) you can examine the packets going between the browser and the back end servers. I don't have time to look at this right now but if you're comfortable programming or with web stuff the packets going back and forth are pretty easy to understand.

**Comment by developersteve** [comment]
Yup will dig around it a bit and drop a PR in
